change/@uifabric-example-data-2020-04-06-10-28-34-caperez-filemap-mar2020-fab7.json

```diff
@@ -0,0 +1,9 @@
+{
+  "type": "patch",
+  "comment": "removing more references to wrong icon-ppt.png icon in wrong folder",
```

Comment:
Can you check the messages for each change file and make sure they properly correspond to the changes in that particular package?